src/components/scenes/Staking/StakeOverviewScene.tsx

```diff
@@ -67,35 +68,41 @@ const StakeOverviewSceneComponent = (props: Props) => {
   const [rewardAllocations, setRewardAllocations] = React.useState<PositionAllocation[]>([])
   const [unstakedAllocations, setUnstakedAllocations] = React.useState<PositionAllocation[]>([])
 
-  // Background loop to force fetchStakePosition updates
-  const [updateCounter, setUpdateCounter] = React.useState<number>(0)
+  const updatePosition = React.useCallback(() => {
```

Comment:
Remove this `useCallback` wrapper, since this function is just used by effects, and not passed as a prop. We only need to memoize when props are involved. In fact, you could probably just inline the `dispatch` call into the two effects, since it's simple enough.